main.go

```diff
@@ -18,11 +21,19 @@ import (
 )
 
 var (
-	tempJsonFileName = "csv-report-temp.json"
-	templateFileName = "csv.tpl"
+	tempJsonFileName     = "csv-report-temp.json"
+	tempTemplateFileName = "csv-report-template-temp.tpl"
+	templateFileName     = "csv.tpl"
+	availableFields      = []string{"Target", "Vulnerability Class", "Target Type", "Vulnerability ID", "Severity", "PackageName", "Installed Version", "Fixed Version", "Title", "Description", "Resolution", "Reference", "Additional Reference", "CVSS V3 Score", "CVSS V3 Vector"}
+	availableFlags       = []string{"--csv-result", "--delimiter", "--include", "--exclude"}
```

Comment:
let's use more specific names for the flags: `--csv-delimiter`, `--csv-include`, `--csv-exclude`